docs/operator/helm-getting-started.md

```diff
@@ -0,0 +1,477 @@
+
+Deploy the LogsInstance into your cluster:
+
+```yaml
+apiVersion: monitoring.grafana.com/v1alpha1
+kind: LogsInstance
+metadata:
+  name: primary
+  namespace: default
+  labels:
+    agent: grafana-agent-logs
+spec:
+  clients:
+  - url: your_remote_logs_URL
+    basicAuth:
+      username:
+        name: primary-credentials-logs
+        key: username
+      password:
+        name: primary-credentials-logs
+        key: password
+
+  # Supply an empty namespace selector to look in all namespaces.
+  podLogsNamespaceSelector: {}
+  podLogsSelector:
+    matchLabels:
+      instance: primary
+```
+
+This LogsInstance will pick up PodLogs resources with the `instance: primary` label. Be sure to set the Loki URL to the correct push endpoint (for Grafana Cloud, this will be something like `logs-prod-us-central1.grafana.net/loki/api/v1/push`, however you should check the Cloud Portal to confirm). 
+
+Also note that we are using the `agent: grafana-agent-logs` label here, which will associate this LogsInstance with the GrafanaAgent resource defined in Step 2. This means that it will inherit requests, limits, affinities and other properties defined in the GrafanaAgent custom resource. 
+
+Create the Secret for the LogsInstance resource:
+
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: primary-credentials-logs
+  namespace: default
+stringData:
+  username: 'your_username_here'
+  password: 'your_password_here'
+```
+
+If you're using Grafana Cloud, you can find your hosted Loki endpoint username and password in the [Grafana Cloud Portal](https://grafana.com/profile/org ). You may wish to base64-encode these values. In this case, please use `data` instead of `stringData`. 
+
+Finally, we'll roll out a PodLogs resource to define our logging targets. Under the hood, Agent Operator will turn this into Agent config for the logs subsystem, and roll it out to the DaemonSet of logging agents.
+
+The following is a minimal working example which you should adapt to your production needs:
+
+```
+apiVersion: monitoring.grafana.com/v1alpha1
+kind: PodLogs
+metadata:
+  labels:
+    instance: primary
+  name: kubernetes-pods
+  namespace: default
+spec:
+  pipelineStages:
+    - docker: {}
+  namespaceSelector:
+    matchNames:
+    - default
+  selector:
+    matchLabels: {}
+```
+
+This tails container logs for all Pods in the `default` Namespace. You can restrict the set of Pods matched by using the `matchLabels` selector. You can also set additional `pipelineStages` and create `relabelings` to add or modify log line labels. To learn more about the PodLogs spec and available resource fields, please see the [PodLogs CRD](https://github.com/grafana/agent/blob/main/production/operator/crds/monitoring.grafana.com_podlogs.yaml).
+
+Under the hood, the above PodLogs resource will generate the following Agent logs `scrape_configs`:
+
+```
```

Comment:
I think this section might be a little overwhelming. Do we want to just explain the labels that will be added to targets instead of showing the specific generated relabel configs?